modules/somalier/extract/meta.yml

```diff
@@ -0,0 +1,63 @@
+name: "somalier_extract"
+description: Somalier can extract informative sites, evaluate relatedness, and perform quality-control on BAM/CRAM/BCF/VCF/GVCF or from jointly-called VCFs
+keywords:
+  - relatedness
+  - QC
+  - BAM/CRAM
+  - VCF/GVCF
+  - ancestry
+  - identity
+  - kinship
+  - informative sites
+  - family
+tools:
+  - "somalier":
+      description: "Somalier can extract informative sites, evaluate relatedness, and perform quality-control on BAM/CRAM/BCF/VCF/GVCF or from jointly-called VCFs"
+      homepage: "https://github.com/brentp/somalier"
+      documentation: "https://github.com/brentp/somalier/blob/master/README.md"
+      tool_dev_url: "https://github.com/brentp/somalier"
+      doi: "https://doi.org/10.1186/s13073-020-00761-2"
+      licence: "MIT License"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
```

Comment:
Givne this can accept many different types, why not call it `input` and `input_index` (or similar)